src/libraries/Microsoft.Extensions.DependencyInjection/tests/TrimmingTests/Microsoft.Extensions.DependencyInjection.TrimmingTests.proj

```diff
@@ -3,6 +3,7 @@
 
   <PropertyGroup>
     <AdditionalProjectReferences>Microsoft.Extensions.DependencyInjection</AdditionalProjectReferences>
+    <IsPackable>false</IsPackable>
```

Comment:
Just curious, why is this necessary?

Reply:
```
  <!-- Remove when https://github.com/NuGet/Home/issues/10405 is implemented and consumed. -->
  <Target Name="ErrorForMissingPackageDescription"
          Condition="'$(IsPackable)' == 'true' and '$(PackageDescription)' == ''">
    <Error Text="Required property 'PackageDescription' is missing for $(MSBuildProjectName)." />
  </Target>

```

This target gets run while running the linker tests and it fails as we moved the package Description to the src project.

Reply:
See my other comment in the PR. These projects should not be marked as `<IsPackable>true</IsPackable>`.